### PR TITLE
Left navigation transition

### PR DESCRIPTION
**Pull request by ddouglasz**
#### Summary

Left navigation transition

### CODE REVIEW

**Comment by ddouglasz** [comment]
@FilPob I faced some challenges with the transition for when the sidebar collapses. The transition works fine for the sidenav when it collapses but since the menu item title is basically made to disappear, there is hardly a way to make the transition happen before the item disappears, thus causing a small glitch in the collapse transition. We took a look at it and we concluded to ask you what you think about it. 

Currently:

https://github.com/commercetools/merchant-center-application-kit/assets/19975842/8d320bfc-19ce-451f-9fcb-a71dc37c9a05



Slowed down version sp you can see how the navbar transitions but the item title would not be captured:

https://github.com/commercetools/merchant-center-application-kit/assets/19975842/72dc41d0-1895-43ba-8e02-a25634a8f469

**Comment by FilPob** [comment]
@ddouglasz thanks for the effort! yeah it is a little glitchy, but nothing too critical that should block the release. I think we can live with that in the phase 1

**Comment by ddouglasz** [comment]
> @ddouglasz thanks for the effort! yeah it is a little glitchy, but nothing too critical that should block the release. I think we can live with that in the phase 1

Alright, great. My thought exactly, we could dig deeper in phase 2. 
Thank you.

File: packages/application-shell/src/components/application-shell/application-shell.spec.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ const renderApp = (ui, options = {}) => {
     // Wait for the loading navbar to disappear. Instead of using `waitForElementToBeRemoved`,
     // which seems not stable enough, we wait to find the "Support" link, which is present
     // when the navbar is loaded.
-    await screen.findByText('Support');
+    await screen.findByRole('navigation');
   };
```
Comment (kark) [inline packages/application-shell/src/components/application-shell/application-shell.spec.js:130]: Perhaps we could edit the comment as well.

File: packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css [hunk 2]
```diff
@@ -15,6 +15,24 @@
...
+}
+
+@keyframes hidden {
+  from {
+    opacity: 1;
+  }
+  to {
+    opacity: 0;
+  }
+}
```
Comment (kark) [inline packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css:34]: I don't think it's used anywhere

**Review by kark — APPROVED** [review]
Big thanks, Douglas! Great job. It looks like we're all set to proceed 🙌
Just a friendly reminder to make sure the changeset is incorporated

File: packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css (hunk 2 above)
Comment (ddouglasz) [reply packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css]: Thank you,
All cleaned up here: https://github.com/commercetools/merchant-center-application-kit/pull/3237/commits/8dc2680475702e90085f24a037efa476469e5290

**Review by chloe0592 — APPROVED** [review]
@ddouglasz thank you for your efforts! Looks good to me, we can work on the details in the phase 2. ☺️

**Pull request merged**